packages/webapp/pages/onboarding.tsx

```diff
@@ -201,12 +220,11 @@ export function OnboardPage(): ReactElement {
       });
     }
 
-    return router.replace({
-      pathname: '/',
-      query: {
-        ua: 'true',
-      },
-    });
+    if (appExperiment && activeScreen !== OnboardingStep.AndroidApp) {
+      return setActiveScreen(OnboardingStep.AndroidApp);
+    }
```

Comment:
It will conflict with @ilasw code, we need to be careful which order they come in.

Reply:
We need to check with product team on how we want these onboarding UIs to be displayed.